test/deployment/pom.xml

```diff
@@ -26,10 +26,6 @@ under the License.
     <version>1.0-SNAPSHOT</version>
     <name>test-deployment</name>
     <url>http://maven.apache.org</url>
-    <properties>
```

Comment:
Does test-deployment work without it?

I think it is needed, otherwise `maven` will try default to compiling it with Java 7.

Reply:
yes that's the point, @lolski . We do want Client Java to work with *any* Java version, and NOT just just 8+. I previously had to add this `properties` section to enforce Java+ because of the `Grakn` interface having a static method `Grakn.client(..)`, which is a Java 8 feature. We've removed that now, so users open client with `Grakn.Client client = new GraknClient(...);`. So we now we're backwards compatible and should remove the `properties` section`.

Reply:
Alright makes sense